app/main/views/send.py

```diff
@@ -1039,6 +1039,8 @@ def send_notification(service_id, template_id):
             ".view_job",
             service_id=service_id,
             job_id=upload_id,
+            from_job=upload_id,
+            notification_id=notifications["notifications"][0]["id"],
```

Comment:
Why are we putting this back? It was removed in one of Bev's PRs.  If it is there, we get intermittent errors in new relic.

Reply:
@A-Shumway42 did this somehow make its way back in?  If you merge with `main` does it go away, or was it intentionally put back for a reason?

Reply:
@ccostino I'll pull main in this branch and see if it goes away. I recall pulling main at the time before pushing. Will try again.

Reply:
@ccostino  After pulling main. It is still there. Do we want to remove this code so it's not placed back in?

Reply:
@A-Shumway42 yes please!  It looks like this last bit here with the `return_redirect` should look like this:

```python
    return redirect(
        url_for(
            ".view_job",
            service_id=service_id,
            job_id=upload_id,
            # used to show the final step of the tour (help=3) or not show
            # a back link on a just sent one off notification (help=0)
            help=request.args.get("help"),
        )
    )
```

Reply:
Either way, check your code against what's in `main` right now and make sure it matches (unless of course it's something you need as a part of this PR).  Thank you!